src/js/appbar.js

```diff
@@ -2,7 +2,7 @@
     Wood.AppBar = Marionette.LayoutView.extend({
       tagName: "appbar",
       attributes: {
-        class: 'appbar',
+        class: 'wood appbar',
```

Comment:
I think how I am going to solve this namespace issue is through the tagName
wood-appbar
wood-button
.. this is how polymer and react do it... namespacing through the component name